docassemble/juvenilesealing/data/questions/juvenile-sealing-text-10.yml

```diff
@@ -90,58 +90,30 @@ content: |
   ${ docket_sheets_blurb } ${ read_more_docket_sheets }
 ---
 id: certified_copies_page_text_question
-field: about_docket_sheets
+field: request_docket_sheets
 # TODO: Are there courthouse instructions online for requesting
 #    docket sheets? A: No
-# [text](${ url_action('about_docket_sheets') })
+# [text](${ url_action('request_docket_sheets') })
 terms:
   a copy of your Massachusetts criminal records: ${ records_popover }
   your Massachusetts CORI report and on your juvenile records: ${ records_popover }
-question: About Docket Sheets
+question: How to Get Docket Sheets
 subquestion: |
-  ### What are docket sheets?
-
-  ${docket_sheets_blurb} Docket
-  sheets are papers from the courthouse that say
-  what happened in a case every time you went
-  to court for that case.
-  
-  ### What's different about a certified copy of the docket sheets?
-
-  A certified copy of the sheets
-  will have a special mark on it that looks like the
-  courthouse's seal. It'll either be a colored stamp or
-  an area where the paper is raised up in the shape of the
-  seal.
-
-  ### Why can sealing be a problem?
-
-  If you seal your records you won't be able to get
-  certified copies of your docket sheets until you
-  unseal them again. That can sometimes take a long time.
-  If you need to show your docket
-  sheets to a judge at an immigration hearing or if you
-  apply for some kinds of jobs, it might take too long
-  to unseal your records
-  and get those docket sheets.
-
   ## Getting certified copies
 
+  Having
```

Comment:
Just for later - where did this text block come from? It looks familiar, but I can't find it elsewhere.

Reply:
I just realized I can comment on multiple lines. Will do that.